.gitignore

```diff
@@ -1,7 +1,9 @@
+!openedx_webhooks/lib/
```

Comment:
Oh, because your .gitignore_global ignores "lib".  That seems really dangerous as a global setting.  I use the global gitignore only for unusual things that I personally add to repos.